packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1798,6 +1800,15 @@ class EditableText extends StatefulWidget {
   /// {@endtemplate}
   bool get selectionEnabled => enableInteractiveSelection;
 
+  /// {@template flutter.widgets.editableText.selectAllOnFocus}
+  /// Whether or not this field should highlight all text when gaining focus on
+  /// web or desktop.
```

Comment:
"highlight" => "select". Just to make sure that we're consistent with our wording.

Reply:
In another comment in this review I suggested that this parameter should apply equally to all platforms. If so then you should also remove "on web or desktop" here.